ui/src/app/pages/signed-in/component.css

```diff
@@ -18,15 +18,12 @@ body {
 }
 
 .background-image {
-  background-image: url('/assets/images/hex-background@2x.png');
-  background-repeat: no-repeat;
+  background-image: url('/assets/images/BG-Pattern.png');
+  background-size: 80px; /* half the size of the image */
```

Comment:
I'd there a way to refer to the one side directly and have the calculation happen here?

Reply:
Not that I could find from a quick search on google. Do you think one exists?

Reply:
Maybe, but I think you may have to hand-enter the image dimensions. https://stackoverflow.com/a/11841812

I was just curious. You don't need to spend a ton of time tracking it down. I really haven't done much css at all.